### DISCUSSION TITLE
Forge WebUi needs an icon!

### DISCUSSION THREAD

**Discussion by sutpidic** [lllyasviel/stable-diffusion-webui-forge#1511] [Ideas]
_Style is copied from [ParityError](https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/2901)._ 

Can't live without icons so realising there's no logos of Forge I made it for two last versions. And I'd love to attach a .psd and .ico files, but only png is allowed. As a bonus I made LoRAs and CNets icons for models folder in style of Stability logo. So here it is:
1. Go to https://convertio.co/ or https://www.freeconvert.com/ and convert png to ico.
2. Make a shortcut of your Forge folder or/and .bat file and hide it in settings. 
3. Change an icon of shortcut in settings of it.

![forgeblueflux](https://github.com/user-attachments/assets/6e05f808-09cd-4e66-a43e-242c294927e6)
![forgeblue](https://github.com/user-attachments/assets/2dd6f478-e67a-4cc3-9b69-8a1c7728e06a)
![forge](https://github.com/user-attachments/assets/7323ee36-4bc3-4be5-ab37-9718caf984bc)
![forge2 0](https://github.com/user-attachments/assets/6946c13b-856d-4f09-8f40-d285a7ef8002)
![image](https://github.com/user-attachments/assets/9ebdd0cf-02e2-497d-b87b-f98555396d23)
![2024-08-26_14-46-33](h
![controlnet models](https://github.com/user-attachments/assets/6fe96b78-ee44-488c-b4e5-4e0f12c6ba78)
![lora models](https://github.com/user-attachments/assets/e3d8a97b-3ede-4f65-b7f6-bd323d85cd5c)
ttps://github.com/user-attachments/assets/02d367bd-5057-4f03-94bc-49dbdd8bbd6f)
![2024-08-26_14-46-33](https://github.com/user-attachments/assets/740b9f0c-98bb-4282-9f2a-461f73d21e5c)
![2024-08-26_15-13-00](https://github.com/user-attachments/assets/4868351e-f6c6-45b7-ad95-e5561e9d0ff7)

**Comment by sutpidic** [comment]
Thanks for likes! Really aprecciate that

**Comment by mrhartsclube** [comment]
<img width="513" height="513" alt="ForgeUI" src="https://github.com/user-attachments/assets/231a0bb9-cb85-47a3-8083-cc6d29f24b7a" />

This is what I made and use in my start menu (an anvil for the word Forge and circuit lines for AI and the word F as a whole), here is a partial preview:

<img width="690" height="598" alt="image" src="https://github.com/user-attachments/assets/1bb2a054-442a-41bb-838b-1b0bcb1539d9" />

I'm in making of a growing collection of these tiles 😊 I might share it someday but for now feel free to use the Forge tile image as icon or anything, also feel free to do any changes you want or use it anywhere you want 😃
Here is the full collection preview until now:

<img width="1072" height="1006" alt="image" src="https://github.com/user-attachments/assets/983bfb35-73d4-4120-ba33-4c9b626e90d5" />
